src/pages/home/ReportScreen.js

```diff
@@ -273,6 +279,15 @@ function ReportScreen({
         [route],
     );
 
+    useEffect(() => {
+        if (prevIsDraggingOver === isDraggingOver) {
+            return;
+        }
+        if (isDraggingOver && closePopover) {
+            closePopover();
+        }
+    }, [isDraggingOver, closePopover, prevIsDraggingOver]);
```

Comment:
Do we need this block in the context [of the new call](https://github.com/Expensify/App/pull/31154/files#r1392491208)?